docs/diagrams/content_block_manager_models.puml

```diff
@@ -17,6 +16,7 @@ together {
         details
         document_id
         state
+        title
```

Comment:
Good catch! I would have totally forgotten about this! 🤦

Reply:
I think it's all a bit out of date but at least this field won't be 😆